src/main/java/software/amazon/awssdk/crt/checksums/Crc.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+package software.amazon.awssdk.crt.checksums;
+
+import software.amazon.awssdk.crt.CrtResource;
+
+public abstract class Crc extends CrtResource {
```

Comment:
Don't make this a CrtResource.  CrtResource is mostly for instantiable objects that have an associated native object where the lifetimes need to be delicately managed.